eslint.config.mjs

```diff
@@ -382,6 +382,18 @@ export default [
         {
           name: 'semver',
           message: 'Please use semifies instead.'
+        },
+        {
+          name: 'get-port',
+          message: 'Please listen on port 0 instead.'
+        },
+        {
+          name: 'rimraf',
+          message: 'Please use fs.rm(path, { recursive: true }) instead.'
```

Comment:
Technically the equivalent is

```suggestion
          message: 'Please use fs.rm(path, { recursive: true, force: true }) instead.'
```

I've created a follow-up PR: https://github.com/DataDog/dd-trace-js/pull/6592